extension/js/common/settings.ts

```diff
@@ -380,7 +381,7 @@ export class Settings {
       await Ui.modal.info(`Reload after logging in.`);
       return window.location.reload();
     }
-    const authRes = await GoogleAuth.newOpenidAuthPopup({ acctEmail });
+    const authRes = await BrowserMsg.send.bg.await.reconnectAcctAuthPopup({ acctEmail });
```

Comment:
Before when calling `GoogleAuth.newOpenidAuthPopup` directly, extension couldn't handle auth window because it's running via content script.
To fix that, we have to send event to background page.

Reply:
Before, when user taps `OK` button, it indeed shows google auth popup. But couldn't handle result after that.
![image](https://user-images.githubusercontent.com/77344191/202180216-4e353ede-f23a-4051-aeb7-b50620aafa4a.png)

Reply:
Got it. Thank you for adding additional check